datadog_cluster_agent/metadata.csv

```diff
@@ -17,6 +17,7 @@ datadog.cluster_agent.admission_webhooks.reconcile_errors,gauge,,,,Number of rec
 datadog.cluster_agent.admission_webhooks.reconcile_success,gauge,,success,,Number of reconcile successes per controller,0,datadog_cluster_agent,admission webhooks reconcile success,
 datadog.cluster_agent.admission_webhooks.response_duration.count,count,,,,Webhook response duration count,0,datadog_cluster_agent,webhook response duration count,
 datadog.cluster_agent.admission_webhooks.response_duration.sum,count,,second,,Webhook response duration sum,0,datadog_cluster_agent,webhook response duration sum,
+datadog.cluster_agent.admission_webhooks.validation_attempts,gauge,,,,Number of pod validation attempts by validation type,0,datadog_cluster_agent,admission webhooks validation attempts,
 datadog.cluster_agent.admission_webhooks.webhooks_received,gauge,,,,Number of mutation webhook requests received,0,datadog_cluster_agent,admission webhooks received,
```

Comment:
i think there was a change to the description of one of the metrics in the PR here https://github.com/DataDog/datadog-agent/pull/28512/files#diff-6ceed6d8bf3c9f5446bf0b5fc8ec56be0883cd12add905b301ee4402e5e187feR53-R59
```suggestion
datadog.cluster_agent.admission_webhooks.webhooks_received,gauge,,,,Number of webhook requests received,0,datadog_cluster_agent,admission webhooks received,
```

Reply:
and nit: same change in the `tests/fixtures/metrics.txt`

Reply:
Thank you for catching the changes in `webhooks_received`!